tf2_ros/CMakeLists.txt

```diff
@@ -82,45 +67,31 @@ target_link_libraries(static_transform_publisher
   ${PROJECT_NAME}
   static_transform_broadcaster_node
 )
```

Comment:
This executable also depends on tf2 and rclcpp; shouldn't it have dependencies on those?

Reply:
In https://github.com/ros2/geometry2/pull/493/commits/7dac8dd51997b0d0892a0e527e12c5f738b98d24 , https://github.com/ros2/geometry2/pull/493/commits/2a9df115fcde75dbd54ddc9347e55d66c1dbedd7 , and https://github.com/ros2/geometry2/pull/493/commits/e78a34d4724545ce557b9cf568d66819dfcd6c6d I double checked the dependencies, and applied the method we came up with in #496 of documenting some dependencies of tests instead of linking them.